src/adapter/src/coord/sequencer/inner/subscribe.rs

```diff
@@ -281,8 +281,6 @@ impl Coordinator {
             }
         }
 
-        self.store_transaction_read_holds(ctx.session(), read_holds);
```

Comment:
I decided to pass the read holds through the `Stage*` results, rather than adding them to the txn holds, as that is more explicit and we don't have to `expect` their existence later.